framework/src/modules/chain/blocks/blocks.js

```diff
@@ -32,6 +32,7 @@ const EVENT_NEW_BLOCK = 'EVENT_NEW_BLOCK';
 const EVENT_DELETE_BLOCK = 'EVENT_DELETE_BLOCK';
 const EVENT_BROADCAST_BLOCK = 'EVENT_BROADCAST_BLOCK';
 const EVENT_NEW_BROADHASH = 'EVENT_NEW_BROADHASH';
+const EVENT_FORK_DETECTED = 'EVENT_FORK_DETECTED';
```

Comment:
This is not correct as this event is only triggered in the case when a block with priority has detected. Maybe something like: `EVENT_PRIORITY_CHAIN_DETECTED` would be a more accurate name for this event?

Reply:
@SargeKhan  I asked earlier for that name yesterday, I wish you had told this earlier.